pkg/instrumentation/instrumentation.go

```diff
@@ -17,7 +17,7 @@ package instrumentation
 import (
```

Comment:
Minor nit to commit `pkg/http: move out....`
Please add to the commit body the reasoning why we are moving this, perhaps something like
`Moving the directory in order to allow it to be used in other parts of the nri-resmgr`
or something like that.